pyth-sdk-terra/README.md

```diff
@@ -0,0 +1,71 @@
+# Pyth SDK Terra
+
+This crate provides utilities for reading price feeds from the [pyth.network](https://pyth.network/) oracle on the Terra network.
+The crate includes a library for reading and using Pyth data feeds in Terra.
+
+## Usage
+
+### Read price
+ 
+For reading the price you just need to call `query_price` function within your contract.
+
+```rust
+// let price_id = Binary::from(b"xyz...");
+
+// Contract address is defaulted to the mainnet address, for testnet use `testnet` feature flag.
+let pyth_contract_addr = pyth_sdk_terra::CONTRACT_ADDR;
+
+let price_feed: PriceFeed = query_price_info(deps.querier, contract_addr, price_id).unwrap().price_feed;
```

Comment:
imo this should be called 'get_price_feed'

also fix `query_price` above

Reply:
I think query is used for terra. I renamed it to `query_price_feed` and I changed return struct wrapper as `PriceFeedReponse`. I can return PriceFeed directly in this query too. If someone needs other fields in PriceFeed response they can query manually.